command/identify.py

```diff
@@ -1,13 +1,15 @@
 import yaml
 
 def get_operator_artifact_type(operatorArtifactString):
+    """get_operator_artifact_type takes a yaml string and determines if it is one of the expected bundle types
```

Comment:
Could you change this to mention that it checks for the expected CSV, CRD, or Package type?

Reply:
Updated in latest commit.